src/pages/filter/ui/panels/range-panel.tsx

```diff
@@ -107,20 +116,13 @@ export const RangePanel = observer((): ReactElement => {
   }
 
   useEffect(() => {
+    if (!min && !max) return
```

Comment:
what if zero?

Reply:
In this case we will have <Numeric filter> == 0 expression at decision tree. As for FR, it's not ready for complex expressions yet

Reply:
ok